bin/asciibinder

```diff
@@ -104,6 +115,23 @@ Description:
   Creates a new, bare AsciiBinder repo in the specified directory.
 EOF
     end
+  when "clone"
+    Trollop::options do
+      banner <<-EOF
+Usage:
+  #$0 clone <options> <git_repo_url>
+
+Description:
+  Clones an existing AsciiBinder repo to the current directory.
+  Under the default behavior, AsciiBinder will attempt to set up
+  tracking branches based on the contents of _distro_map.yml,
+  but this can be suppressed (see Options).
+
+Options:
+EOF
+      opt :nobranches, "Don't attempt to create tracking branches after cloning.", :type => :bool, :default => false
```

Comment:
Much obliged. I didn't like implementing it that way; your suggestion is much better.